ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/statetransition/epoch/AbstractEpochProcessor.java

```diff
@@ -122,7 +125,7 @@ protected void processEpoch(final BeaconState preState, final MutableBeaconState
     processSyncCommitteeUpdates(state);
 
     if (beaconStateAccessors.isInactivityLeak(state)) {
-      LOG.info("Beacon chain is in activity leak");
+      loggerThrottler.invoke(currentEpoch, (log) -> log.info("Beacon chain is in activity leak"));
```

Comment:
tests or it didnt happen...

Reply:
ouch forgot to add the file to the commit

Reply:
i'd be happy with just a test showing a logThrottler in use with this mechanism.. doesnt have to be in-situe testing...